From 4ce9896c6ac42d450d2056ee8d700b9f6dce8e2d Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Wed, 11 Jan 2012 11:42:09 +0100
Subject: [PATCH] Revert "fast_merge: fix mis-merge of first module's strings"

This is not enough, because the file is still wrong.

This reverts commit 47a8454f5f76030203ce6458cf63d4348bdacffe.
---
 l10ntools/scripts/fast_merge.pl |   10 ++++------
 1 files changed, 4 insertions(+), 6 deletions(-)

diff --git a/l10ntools/scripts/fast_merge.pl b/l10ntools/scripts/fast_merge.pl
index 73b824e..5dc63cf 100644
--- a/l10ntools/scripts/fast_merge.pl
+++ b/l10ntools/scripts/fast_merge.pl
@@ -91,7 +91,7 @@ while( hasLines() )
 }
 if( $#current+1 ne 0 )
 {
-    ( $path , $localize_file ) = make_paths($current[ 0 ]->module);
+    ( $path , $localize_file ) = make_paths();
     add_to_buffer();
     write_buffer( $path , $localize_file );
 }
@@ -240,8 +240,7 @@ sub hasLines
 
 sub make_paths
 {
-    my $module = shift ;
-    my $localizeFile = $merge_dir."\\".$module."\\".$current[ 0 ]->file;
+    my $localizeFile = $merge_dir."\\".$current[ 0 ]->module."\\".$current[ 0 ]->file;
     my $path = getDir( $localizeFile );
     $path =~ s/\\/\//g;
 
@@ -252,9 +251,8 @@ sub make_paths
 sub write_lines
 {
     if( $first_run ){
-        my $module = $current[ 0 ]->module;
         add_to_buffer();
-        my( $path , $localize_file ) = make_paths($module);
+        my( $path , $localize_file ) = make_paths();
         $last_path = $path;
         $last_localize_file = $localize_file;
         mkpath $path;
@@ -264,7 +262,7 @@ sub write_lines
     else
     {
         return , if ( $#current+1 eq 0 );
-        my( $path , $localize_file ) = make_paths($current[ 0 ]->module);
+        my( $path , $localize_file ) = make_paths();
         if( $path eq $last_path )
         {
             add_to_buffer();
-- 
1.7.7.5

